### PR TITLE
Introducing the OpenGraphParser, londis_gb and premier_gb example use…

### PR DESCRIPTION
**Pull request by mjoe999**
…s give +5K stores

Makes its debug with the Londis and Premier GB symbol group convenience stores.

### CODE REVIEW

File: locations/spiders/londis_gb.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "brand_wikidata": "Q21008564",
+        "country": "GB",
+    }
+    allowed_domains = ["londis.co.uk"]
+    sitemap_urls = ["https://www.londis.co.uk/sitemap.xml"]
+    sitemap_rules = [("/our-stores/", "parse_store")]
+    download_delay = 1.0
+
+    def parse_store(self, response):
+        return OpenGraphParser.parse(response)
```
Comment (iandees) [inline locations/spiders/londis_gb.py:19]: This should `yield`, not `return`.

File: locations/spiders/premier_gb.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "brand_wikidata": "Q7240340",
+        "country": "GB",
+    }
+    allowed_domains = ["premier-stores.co.uk"]
+    sitemap_urls = ["https://www.premier-stores.co.uk/sitemap.xml"]
+    sitemap_rules = [("/our-stores/", "parse_store")]
+    download_delay = 1.0
+
+    def parse_store(self, response):
+        return OpenGraphParser.parse(response)
```
Comment (iandees) [inline locations/spiders/premier_gb.py:19]: Same here.

File: locations/open_graph_parser.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from locations.items import GeojsonPointItem
+from locations.dict_parser import DictParser
+
+
+class OpenGraphParser(object):
```
Comment (iandees) [inline locations/open_graph_parser.py:5]: Can you please add a test for this?

File: locations/spiders/premier_gb.py (hunk 2 above)
Comment (mjoe999) [reply locations/spiders/premier_gb.py]: Done

File: locations/open_graph_parser.py (hunk 3 above)
Comment (mjoe999) [reply locations/open_graph_parser.py:5]: Done and caught issue.

**Pull request merged**